install/debian/postrm

```diff
@@ -10,6 +10,9 @@ case "$1" in
   purge)
     # Remove all files
     rm -rf /etc/$NAME /var/lib/$NAME /var/log/$NAME
+    # Remove v7 cruft
+    rm -rf /var/lib/fahclient /etc/fahclient /etc/default/fahclient
+    rm -rf /var/run/fahclient.pid
```

Comment:
Since `fahclient.pid` is file, plese drop `-r`.

Reply:
Done. Thanks.